snippet.rb

```diff
@@ -0,0 +1,42 @@
+def fetch_data_from_openweathermap(city_name, api_key)
+  uri = URI(BASE_URL_OPENWEATHERMAP)
+  params = {
+    'q' => city_name,
+    'units' => 'metric',
+    'appid' => api_key
+  }
+  uri.query = URI.encode_www_form(params)
+
+  begin
+    response_openweathermap = Net::HTTP.get(uri)
+    weather_data_openweathermap = JSON.parse(response_openweathermap)
+
+    temperature = weather_data_openweathermap['main']['temp']
```

Comment:
would clarify this variable as for a reviewer it might not be clear what temperature are we talking about there, maybe having this variable named as `current_temperature` would give more clarity?